app/views/movies/show.html.erb

```diff
@@ -0,0 +1,34 @@
+<div class='movie-details'>
+  <img src="<%= @movie.poster %>" class='movie-poster'>
+  <h2><%= @movie.title %></h2>
+  <h4><%= @movie.production%>, <%= @movie.release_date.year %></h4>
+  <p class='watchlists-and-favorites'>
+    <span><%= @movie.watchlisting_users.count %> watchlists</span>
+    <span><%= @movie.favoriting_users.count %> favorites</span>
+    <% if @movie.watchlisting_users.include?(current_user) %>
+      <%= link_to 'Remove from Watchlist', [@movie, @movie.watchlists.find_by(user_id: current_user.id)], method: :delete %>
```

Comment:
Let's not make SQL queries in the view. Move this logic elsewhere. Where would be best?